developer-docs-site/docs/move/book/functions.md

```diff
@@ -327,6 +327,10 @@ fun zero(): u64 { 0 }
 
 Here `: u64` indicates that the function's return type is `u64`.
 
+:::tip
+A function can return an immutable `&` or mutable `&mut` [reference](./references.md) if derived from an input reference. Keep in mind, this means that a function [cannot return a reference to global storage](./references.md#references-cannot-be-stored).
```

Comment:
I wonder if we should mention here that inline functions can actually return reference to global state as well

Reply:
Yeah, we should probably mention inline as well

Reply:
Sounds good. I'll add it to the tip.